requirements-codecoverage.txt

```diff
@@ -1,4 +1,8 @@
 # Use public repo for coverall upload
```

Comment:
This requirements file really includes test libraries and code coverage libraries.

Reply:
Moved 'ipywidgets' & 'plotly' libraries from requirements-codecoverage.txt to requirements.txt